README.md

```diff
@@ -78,13 +71,17 @@ The table below lists the guest operating systems that are covered by the templa
 
 | Guest OS | Meta-template |
 |---|---|
-| Microsoft Windows Server 2012 R2 | [windows](templates/windows.tpl.yaml) |
+| Microsoft Windows Server 2012 R2 | [windows](templates/windows2k12.tpl.yaml) |
+| Microsoft Windows Server 2016 | [windows](templates/windows2k16.tpl.yaml) |
+| Microsoft Windows Server 2019 | [windows](templates/windows2k19.tpl.yaml) |
 | Microsoft Windows 10 | [windows](templates/windows10.tpl.yaml) |
-| Fedora 32 | [fedora](templates/fedora.tpl.yaml) |
+| Fedora 39 | [fedora](templates/fedora.tpl.yaml) |
```

Comment:
I think it is a bit early to support Fedora 39

Reply:
It is good to be prepared :D